### PR TITLE
Restricted MVT serving

### PR DESCRIPTION
**Pull request by chrstnbwnkl**
Adds a new tutorial on restricted access to geodata via Mapbox Vector Tiles. 

I'm not quite sure yet that this is ready: the last chunk of JS is quite long, but I feel like it needs to be in the tutorial, but on the other hand, there's little to explain, I think the most important parts are explained in the tutorial text. What do you think @nilsnolde ?

### CODE REVIEW

**Review by nilsnolde — CHANGES_REQUESTED** [review]
didn't go all the way, I stopped more or less before the JS part, because I wouldn't be able to run it in its current state. there's quite a few problems with the python code, mostly imports missing which is not so painful, but I have little idea what `Tile` is supposed to be..

generally it's cool functionality and the full experience just needs some JS code and a few more lines of python. I do think the code blocks could use a few more in-code comments what's happening, also in the JS code:)

File: webservices/fastapi/fastapi_auth_vector_tiles.md [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+
+# ...
+
+@vectortile_app.get("/adresses/{z}/{x}/{y}/")
+async def get_tile(
+        request: Request,
+        auth_info: Authorizer = Depends(Authorizer()),
+        tile: Tile = Depends(tile_params),  # FastAPI magic: receives x/y/z
+):
+    tms = morecantile.tms.get("WebMercatorQuad")
```
[the remark this answers is not in the document]
Comment (chrstnbwnkl) [reply webservices/fastapi/fastapi_auth_vector_tiles.md:305]: Usually you're right, but in the context of this tutorial, I think it's actually better to leave the code where it makes sense to explain it.

[the remark this answers is not in the document]
Comment (nilsnolde) [reply webservices/fastapi/fastapi_auth_vector_tiles.md:305]: haha true!

File: webservices/fastapi/fastapi_auth_vector_tiles.md [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+                jwtoken,
+                "SUPER_SECRET_KEY",
+                algorithms=["HS256"],
+            )
+            return TokenPayload(**payload)
+        except Exception:
+            return False
+
+    def get_user_info(self):
+        return get_user_info(self.user)
```
[the remark this answers is not in the document]
Comment (chrstnbwnkl) [reply webservices/fastapi/fastapi_auth_vector_tiles.md:230]: this was on purpose as well, to make clear that the method just calls a function internally that actually holds the logic

**Review by nilsnolde — APPROVED** [review]
jep thanks, looks good! didn't test it again though, but no need if you just did that.

**Pull request merged**